### ISSUE TITLE
FNFBot.apk

### ISSUE THREAD

**Issue by DzGithub2805** [luckydog7/Funkin-android#1244]
If you can port fnf to android, why not port

FNFBot to android 
https://github.com/KadeDev/FNFBot
^
|
|
Source code ^-^

**Comment by netonero** [comment]
but thats a different thing

**Comment by netonero** [comment]
and why the hell did you labelled this "enhancement"

**Comment by DzGithub2805** [comment]
> and why the hell did you labelled this "enhancement"

I want to try :v

**Comment by netonero** [comment]
then again
it needs the app permission where it reads the phone's things

**Comment by JoseGithubFNF** [comment]
> > and why the hell did you labelled this "enhancement"
> 
> I want to try

> then again
> it needs the app permission where it reads the phone's things

Also Just Use A Autoclicker Turn To Into A One Single Button Too Combine And Boom

**Comment by netonero** [comment]
what if you're playing hellclown/expurgation

**Comment by netonero** [comment]
you'd die fast because you're gonna hit the burning notes

**Comment by JoseGithubFNF** [comment]
> you'd die fast because you're gonna hit the burning notes

True! But How Do You Get A Aimbot On Android No Where So Autoclicker are most the same atleast!

**Comment by netonero** [comment]
yeah but that implies that you'd still die real fast because the autoclicker really doesnt hit the notes that well when its a large wall

**Comment by Hurbmoment** [comment]
why are ppl still commenting on this XD its an old issue.

**Comment by Fazefelipe** [comment]
How can you download it
